### ISSUE TITLE
cannot run code on mac

### ISSUE THREAD

**Issue by sahilpatil98** [spyder-ide/spyder#16814]
### What steps will reproduce the problem?

cannot run code on new mac, this error pops up. Even while running code for libraries

### Traceback
```python-traceback
Traceback (most recent call last):
  File "/Users/sahil/opt/anaconda3/lib/python3.8/site-packages/spyder/app/mainwindow.py", line 1767, in execute_in_external_console
    console.switch_to_plugin()
  File "/Users/sahil/opt/anaconda3/lib/python3.8/site-packages/spyder/api/plugins.py", line 406, in switch_to_plugin
    super(BasePluginWidget, self)._switch_to_plugin()
  File "/Users/sahil/opt/anaconda3/lib/python3.8/site-packages/spyder/plugins/base.py", line 284, in _switch_to_plugin
    self.main.maximize_dockwidget()
  File "/Users/sahil/opt/anaconda3/lib/python3.8/site-packages/spyder/app/mainwindow.py", line 1164, in __getattr__
    return super().__getattr__(attr)
AttributeError: 'MainWindow' object has no attribute 'maximize_dockwidget'
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.8.12
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Darwin 21.0.1

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7     :  0.2.1 (OK)
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.29.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.1.0 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=1.0                  :  2.9.6 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.1.1 (OK)
qtpy >=1.5.0                  :  1.10.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  58.0.4 (OK)
sphinx >=0.6.6                :  4.2.0 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.5 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.2.1 (OK)

# Optional:
cython >=0.21                 :  0.29.24 (OK)
matplotlib >=2.0.0            :  3.4.3 (OK)
numpy >=1.7                   :  1.21.2 (OK)
pandas >=1.1.1                :  1.3.4 (OK)
scipy >=0.17.0                :  1.7.1 (OK)
sympy >=0.7.3                 :  1.9 (OK)
```

**Comment by ccordoba12** [comment]
Hey @sahilpatil98, thanks for reporting. This problem is fixed in our latest version (5.1.5). Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda remove spyder
    conda remove python-language-server
    conda update anaconda
    conda install spyder=5.1.5

Some of those commands could give you errors, but that's fine, just continue with the others.

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
